lsp-scala.el

```diff
@@ -60,9 +65,17 @@
   (lsp-send-execute-command "source-scan" ())
   )
 
+(eval-after-load 'lsp
+  '(lsp-register-client
+    (make-lsp-client :new-connection
+		     (lsp-stdio-connection (lambda () (lsp-scala--server-command)))
```

Comment:
There is no need for a lambda here:
```suggestion
           (lsp-stdio-connection 'lsp-scala--server-command)
```

Reply:
Good catch.  I needed that lambda I was referring to the defcustom, but once I split the config into command and args like the other clients, that lambda lost its purpose.